docs/7.x/selinux.md

```diff
@@ -0,0 +1,214 @@
+# SELinux
+
+Starting with version 7, Gravity comes with SELinux support.
```

Comment:
Not sure this needs to be mentioned since we have 7.0 specific docs now.

Also, weren't this going to be backported?